### DISCUSSION TITLE
Why navigation.popTo doesn't have "merge" option?

### DISCUSSION THREAD

**Discussion by gigilibala** [react-navigation/react-navigation#12300] [General]
In version 7.x the new `navigation.popTo()` method doesn't accept the `merge` option like the previous `navigation.navigate()` does. Any special reason for that? Because the `StackAction.popTo()` has the merge option and I'm assuming `navigation.navigate()` uses that internally so why not surface that extra flag in there as well? 

To have this feature, I currently have two options:

* Keep using `navigateDeprecated()` function which is going to be removed in the next major upgrade.
* Switch to using dispatch and StackAction, which in this case I'll lose types in typescript.

It would be nice to have that extra `merge` is surfaced in the `popTo()` method as well if there is no technical hinderance for it.

Thank you.

**Comment by satya164** [comment]
It does https://github.com/react-navigation/react-navigation/blob/c6e7352c1859fcf80ffb8e05c7c3548bb49f5edc/packages/routers/src/StackRouter.tsx#L137
